### ISSUE TITLE
Choice of loss function

### ISSUE THREAD

**Issue by 0xtristan** [manicman1999/StyleGAN2-Tensorflow-2.0#11]
In the styleGAN 2 paper they state using a "non-saturating logistic loss", is there any particular reason you opted for hinge loss in this implementation? 

For reference I believe the original loss functions for config-f are `G_logistic_ns_pathreg` for the generator and `D_logistic_r1` for the discriminator (https://github.com/NVlabs/stylegan2/blob/master/training/loss.py).
